src/main.cpp

```diff
@@ -13,9 +13,8 @@
 
 #include "lammps.h"
 
-#include "accelerator_kokkos.h"
 #include "input.h"
```

Comment:
Are we still using `input.h` explicitly?

Reply:
Yes, it is required. The functionality needed is not available from the library interface and also the behavior of the library interface with exceptions enabled is different and thus would require a more significant rewrite of main.cpp than I currently want to do.